systers_portal/community/tests/test_views.py

```diff
@@ -929,3 +929,54 @@ def test_post_user_permissions_groups(self):
         response = self.client.post(url, data={})
         self.assertEqual(response.status_code, 302)
         self.assertFalse(self.systers_user.is_group_member(group))
+
+
+class CommunitySearchViewTestCase(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='Test', password='foobar')
+        self.systers_user = SystersUser.objects.get(user=self.user)
+        country = Country.objects.create(name='Test Country', continent='AS')
+        location = City.objects.create(name='Test City', display_name='Test City',
+                                       latitude=20, longitude=20,
+                                       country=country)
+        self.community1 = Community.objects.create(name="Foo", slug="foo",
+                                                   order=1, location=location,
+                                                   admin=self.systers_user)
+        self.community2 = Community.objects.create(name="Bar", slug="bar",
+                                                   order=2, location=location,
+                                                   admin=self.systers_user)
+        self.community3 = Community.objects.create(name="Baz", slug="baz",
+                                                   order=3, location=location,
+                                                   admin=self.systers_user)
+
+    def test_search_results(self):
```

Comment:
Could you add one with empty `query`, that should cover the else branch.

Reply:
Ok I am currently working on meetup views. I will make changes by tonight. Will that be ok?